internal/storage/v2/grpc/trace_storage.proto

```diff
@@ -145,7 +151,7 @@ service TraceReader {
   // Edge cases:
```

Comment:
you may want to rephrase L147 to avoid referring to `TracesData` entry as a chunk. In fact, the whole explanation should probably move to TracesChunk type comments.